.circleci/config.yml

```diff
@@ -75,21 +75,37 @@ workflows:
               - "gemfiles/rails_5.2.gemfile"
               - "gemfiles/rails_6.0.gemfile"
               - "gemfiles/rails_6.1.gemfile"
+              - "gemfiles/rails_7.0.gemfile"
```

Comment:
This test matrix is getting pretty big. Should we use this as an opportunity to drop support for old versions of Ruby and Rails?